teps/0096-pipelines-v1-api.md

```diff
@@ -123,19 +131,59 @@ of TEP-0033: Feature Gates, Tekton's policy allows a behavior change to be updat
 A behavior can't be changed from opt-in to opt-out until 9 months after it is introduced to a beta API or 12 months after it is introduced to a stable API.
 This is different from Kubernetes' policy, which would require creating a new API version to change a feature from opt-in to opt-out.
 
-### API Definition
-
 ## Use Cases
 
-### V1 Supported Use Cases
+Most Tekton use cases are supported by features from multiple Tekton projects, meaning that there are relatively few use cases for
+Pipelines as an independent project. The following use cases guide Tekton feature work, and a Pipelines V1 API should stabilize
+the features critical to supporting them.
+
+### Use Cases prioritized for Tekton
+- As an application engineer, I want to run tests when new pull requests are created and automatically
+merge only changes that have passed these tests, so we are always in a releaseable state.
```

Comment:
What do you think of rewording this to - As an application developer, I want to design a continuous integration process for my application. The continuous integration includes running tests (unit/integration/etc) when new pull requests are created. And automatically merge only changes that have passed these tests and collected enough reviews such that we are always in a releasable state.

Reply:
I am leaning towards trying to keep use cases concise because they haven't actually helped with scoping as much as I originally expected. Each of these use cases could be expanded to a large number of features, but I think we should keep "new features we want in v1" as short of a list as possible. I'd prefer to keep as is.